lib/util/fs/index.spec.ts

```diff
@@ -113,57 +137,6 @@ describe('util/fs/index', () => {
     });
   });
 
-  describe('readLocalDirectory', () => {
-    it('returns dir content', async () => {
-      await withDir(
-        async (localDir) => {
-          GlobalConfig.set({
-            localDir: localDir.path,
-          });
-          await writeLocalFile('test/Cargo.toml', '');
-          await writeLocalFile('test/Cargo.lock', '');
-
-          const result = await readLocalDirectory('test');
-          expect(result).not.toBeNull();
-          expect(result).toBeArrayOfSize(2);
-          expect(result).toMatchSnapshot();
-
-          await writeLocalFile('Cargo.lock', '');
-          await writeLocalFile('/test/subdir/Cargo.lock', '');
-
-          const resultWithAdditionalFiles = await readLocalDirectory('test');
-          expect(resultWithAdditionalFiles).not.toBeNull();
-          expect(resultWithAdditionalFiles).toBeArrayOfSize(3);
-          expect(resultWithAdditionalFiles).toMatchSnapshot();
-        },
-        {
-          unsafeCleanup: true,
-        }
-      );
-    });
-
-    it('return empty array for non existing directory', async () => {
-      await withDir(
-        async (localDir) => {
-          GlobalConfig.set({
-            localDir: localDir.path,
-          });
-          await expect(readLocalDirectory('somedir')).rejects.toThrow();
-        },
-        {
-          unsafeCleanup: true,
-        }
-      );
-    });
-
-    it('return empty array for a existing but empty directory', async () => {
-      await ensureLocalDir('somedir');
```

Comment:
do we still have a test for this function? so when running this test file FS should be fully covered?

Reply:
There never was a test but "fake coverage" due to `istanbul ignore next`

I've now changed the method signature to mimic the similar `ensureCacheDir()` function and added tests.